MetaMorpheus/EngineLayer/FdrAnalysis/FdrAnalysisEngine.cs

```diff
@@ -110,7 +110,7 @@ private void DoFalseDiscoveryRateAnalysis(FdrAnalysisResults myAnalysisResults)
                     // really, in this case, we only need to run one or the other (i.e., only peptides or psms are passed in)
                     // but there's no mechanism to pass that info to the FDR analysis engine, so we'll do this for now
                     peptides = psms
-                            .OrderByDescending(p => p)
+                            .OrderByDescending(p => p.FdrInfo.PEP)
```

Comment:
Why are we using linq statements here and above instead of the filtering methods you wrote?